src/form/internal.tsx

```diff
@@ -31,45 +29,28 @@ export default function InternalForm({
 
   return (
     <div {...baseProps} ref={__internalRootRef} className={clsx(styles.root, baseProps.className)}>
-      <FormLayout
-        header={
-          header && <div className={clsx(styles.header, variant === 'full-page' && styles['full-page'])}>{header}</div>
-        }
-        variant={variant}
-      >
-        {children && <div className={styles.content}>{children}</div>}
-        {errorText && (
-          <InternalBox margin={{ top: 'l' }}>
-            <InternalAlert type="error" statusIconAriaLabel={errorIconAriaLabel}>
-              <div className={styles.error}>{errorText}</div>
-            </InternalAlert>
-          </InternalBox>
-        )}
-        {(actions || secondaryActions) && (
-          <div className={styles.footer}>
-            <div className={styles['actions-section']}>
-              {actions && <div className={styles.actions}>{actions}</div>}
-              {secondaryActions && <div className={styles['secondary-actions']}>{secondaryActions}</div>}
-            </div>
+      {header && <div className={styles.header}>{header}</div>}
```

Comment:
Since we're not using the `full-page` class anymore, we should be able to clean [this CSS selector](https://github.com/cloudscape-design/components/blob/main/src/form/styles.scss#L13) to be just `.header`.

Reply:
Nicely spotted! Thank you